### ISSUE TITLE
Issue with glib

### ISSUE THREAD

**Issue by DimiBD** [appenz/kerbal2#1]
Hi,
I just starting with TO2, and your code seems to be a good starting point.

Unfortunatly, It seems to be issue in the glib.
I can't boot KS2

Rebooted in 00:00:01.7258968
ERROR: [glib\transfer.to2(61, 132)] IncompatibleTypes
Cannot BitXor a float with a int
ERROR: [glib\transfer.to2(61, 45)] ArgumentMismatch
Argument d of 'sqrt' has to be a float, but Unit was given
ERROR: [glib\transfer.to2(67, 78)] IncompatibleTypes
Cannot BitXor a float with a float
ERROR: [glib\transfer.to2(67, 18)] IncompatibleTypes
Cannot Mul a float with a Unit
ERROR: [glib\transfer.to2(67, 122)] IncompatibleTypes
Cannot BitXor a Unit with a float
ERROR: [glib\transfer.to2(66, 5)] IncompatibleTypes
Function 'hohmannTime' returns Unit but should return float

I try to make some change, with no succes. I have to read the TO2 specs ;)

The first line in error : 

    sync fn hohmannPhaseAngle(self, alt_start: float, alt_end: float, o: Orbit) -> float = {
        return 180*(1-(1/(2*sqrt(float2))*sqrt((1+(alt_start+o.reference_body.radius)/(alt_end+o.reference_body.radius))^3)))
    }

Maybe each number have to be cast as Float ?

Thx for your help,

DimiBD

**Comment by AndyMt** [comment]
I fixed this in my repo. Replace the ^3 at the end of that line with **3:
```
sync fn hohmannPhaseAngle(self, alt_start: float, alt_end: float, o: Orbit) -> float = {
    return 180*(1-(1/(2*sqrt(float2))*sqrt((1+(alt_start+o.reference_body.radius)/(alt_end+o.reference_body.radius))**3)))
}
```

**Comment by palamedes** [comment]
Thanks Andy!

**Comment by appenz** [comment]
Thanks Andy! I'll see if I can fix this and update this week.

**Comment by appenz** [comment]
Fixed in 0bab58574e71fa69a646061cb7d3cab2e6c120fb.
